interop-testing/build.gradle

```diff
@@ -69,10 +69,18 @@ task reconnect_test_client(type: CreateStartScripts) {
     classpath = jar.outputs.files + configurations.runtime + configurations.tcnative
 }
 
+task stresstest_client(type: CreateStartScripts) {
+    mainClassName = "io.grpc.testing.integration.StressTestClient"
+    applicationName = "stresstest-client"
```

Comment:
I am a bit unsure about the intend of this client. Do the performance and the exported QPS stats matter or is this client intented for testing mostly?